dist/src/main/resources/application-mcp.yaml

```diff
@@ -0,0 +1,23 @@
+camunda:
+  mcp:
+    server:
+      enabled: true
+
+spring:
+  ai:
+    mcp:
+      server:
+        enabled: true
+        name: Camunda 8 Orchestration API MCP Server
+        version: '${project.version}'
+        type: sync
+        protocol: stateless
```

Comment:
Disables any SSE/streaming handling which should be the right fit for our use case.